Documentation/Blazorise.Docs/Pages/Blog/2022-08-31_CSSTipsForDevelopers/Index.md

```diff
@@ -0,0 +1,105 @@
+﻿---
+title: Practical CSS Tips For Developers
+description: 
+permalink: blog/practical-css-tips-for-developers
+canonical: blog/practical-css-tips-for-developers
+image-url: img/blog/2022-08-31/
+image-text:
+author-name:
+author-image;
+posted-on:
+read-time:
+
+
+---
+
+Cascading Style Sheet, the beauty behind the web, friend to some, a devil to many but a necessary evil. Cascading Style Sheets, at their root, are intended to separate the style of a webpage from the content.
+
+CSS files can grow significantly
+
+Fortunately, here are some great CSS tips for developers.
+
+
+## Center anything in three lines of code
+To center one `div` inside another, we make the containing `div` a flex container. Next, set align-items to center to perform centering on the block axis, and justify-content to center to perform centering on the inline axis.
+
+In the future, we may be able to center elements without needing to turn the parent into a flex container, as the Box Alignment properties used here are specified to apply to block layout too.
+
+The CSS `justify-content` property defines how the browser distributes space between and around content items along the main axis of a flex container, and the inline axis of a grid container.
+
+The CSS `justify-items` property defines the default justify-self for all items of the box, giving them all a default way of justifying each box along the appropriate axis.
+
+``` CSS
```

Comment:
Block descriptor must be in lowercase and must not contain spaces.

Reply:
fixed